scripts/commands/checkmission.lua

```diff
@@ -13,7 +13,15 @@ cmdprops =
 
 function onTrigger(player,logId,target)
     
+    local logName;
     logId = tonumber(logId) or _G[logId];
+    if ((type(logId) == "table")) then
+        logName = logId.full_name;
+        logId = logId.mission_log;
+    else
+        local missionAreas = {SANDORIA, BASTOK, WINDURST, ZILART, TOAU, WOTG, COP, ASSAULT, CAMPAIGN, ACP, AMK, ASA, SOA, ROV};
+        logName = missionAreas[logId + 1].full_name;
```

Comment:
They should be included by the require for [missions](https://github.com/ibm2431/darkstar/blob/953cedb579fc20dbd74db3b92f55c115dcecdde8/scripts/globals/missions.lual)